common/components/chrome/MainHeader.jsx

```diff
@@ -31,6 +31,7 @@ class MainHeader extends React.PureComponent<{||}> {
     NavigationDispatcher.dispatch({
       type: 'SET_SECTION',
       section: Section.Landing,
+      url: '/index/?section=Landing'
```

Comment:
why are you passing in this parameter here? Doesn't the `NavigationStore` handle this by itself: 

https://github.com/DemocracyLab/CivicTechExchange/blob/master/common/components/stores/NavigationStore.js#L35

Reply:
We need to pass in parameters for viewing/editing projects, and the navigation system removed any parameters after the ?section= part.  This was the quickest way I saw to add in the url info those pages needed.  

If there's a scheme for doing this that fits better with the Flux paradigm, I'm definitely open to using it.